### PR TITLE
DOCS-314 - Split by field on the filter node

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds information on the new "Split by" field on filter nodes to these sections:
* https://help.sumologic.com/docs/platform-services/automation-service/automation-service-playbooks/#add-a-filter-node-to-a-playbook
* https://help.sumologic.com/docs/cloud-soar/automation/#filter

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-314](https://sumologic.atlassian.net/browse/DOCS-314)

### CODE REVIEW

File: docs/cloud-soar/automation.md [hunk 1]
```diff
@@ -282,6 +282,7 @@ For example:
 
 1. Hover your mouse over an action node and click the **+** button. <br/><img src={useBaseUrl('img/cloud-soar/add-node-button.png')} alt="Add node button" style={{border:'1px solid gray'}} width="200"/> <br/>The available nodes are displayed. <br/><img src={useBaseUrl('img/cloud-soar/add-filter-node.png')} alt="Add filter node" style={{border:'1px solid gray'}} width="500"/>
 1. Click **Filter**. The filter node configuration dialog displays. <br/><img src={useBaseUrl('img/cloud-soar/configure-filter-node.png')} alt="Configure filter node conditions" style={{border:'1px solid gray'}} width="500"/>
+1. (Optional) You can use **Split by** to select an output if it is a list (array) and you want each item in the list to be evaluated separately. Each item in the list is checked against the filter condition, and if the condition is true, the playbook proceeds to the next node. (If you do not use the **Split by** field on an output that is a list, then if the condition is true for only one item in the list, then all the data in the list moves forward to the next node.)
```
Comment (kimsauce) [inline docs/cloud-soar/automation.md:285]:
```suggestion
1. (Optional) Use **Split by** to select an output if it is a list (array) and you want to evaluate each item separately. Each item in the list is checked against the filter condition. If the condition is true for any item, the playbook proceeds to the next node. (If you do not use the **Split by** field on an output that is a list, then if the condition is true for any item in the list, the entire list moves forward to the next node.)
```

File: docs/platform-services/automation-service/automation-service-playbooks.md [hunk 2]
```diff
@@ -183,6 +183,7 @@ A filter node filters results from the preceding action based on the condition y
 1. [Add an action node](#add-an-action-node-to-a-playbook). 
 1. Hover your mouse over an action node and click the **+** button. The available nodes are displayed. <br/><img src={useBaseUrl('img/platform-services/automation-service/automations-add-filter-node.png')} alt="Add filter node" style={{border:'1px solid gray'}} width="500"/>
 1. Click **Filter**. The filter node configuration dialog displays. <br/><img src={useBaseUrl('img/platform-services/automation-service/automations-add-filter-node-conditions.png')} alt="Add filter node conditions" style={{border:'1px solid gray'}} width="500"/>
+1. (Optional) You can use **Split by** to select an output if it is a list (array) and you want each item in the list to be evaluated separately. Each item in the list is checked against the filter condition, and if the condition is true, the playbook proceeds to the next node. (If you do not use the **Split by** field on an output that is a list, then if the condition is true for only one item in the list, then all the data in the list moves forward to the next node.)
```
Comment (kimsauce) [inline docs/platform-services/automation-service/automation-service-playbooks.md:186]:
```suggestion
1. (Optional) Use **Split by** to select an output if it is a list (array) and you want to evaluate each item separately. Each item in the list is checked against the filter condition. If the condition is true for any item, the playbook proceeds to the next node. (If you do not use the **Split by** field on an output that is a list, then if the condition is true for any item in the list, the entire list moves forward to the next node.)
```

**Review by kimsauce — APPROVED** [review]
looks great! had some optional suggestions.

File: docs/cloud-soar/automation.md [hunk 3]
```diff
@@ -282,6 +282,7 @@ For example:
 
 1. Hover your mouse over an action node and click the **+** button. <br/><img src={useBaseUrl('img/cloud-soar/add-node-button.png')} alt="Add node button" style={{border:'1px solid gray'}} width="200"/> <br/>The available nodes are displayed. <br/><img src={useBaseUrl('img/cloud-soar/add-filter-node.png')} alt="Add filter node" style={{border:'1px solid gray'}} width="500"/>
 1. Click **Filter**. The filter node configuration dialog displays. <br/><img src={useBaseUrl('img/cloud-soar/configure-filter-node.png')} alt="Configure filter node conditions" style={{border:'1px solid gray'}} width="500"/>
+1. (Optional) Use **Split by** to select an output if it is a list (array) and you want to evaluate each item separately. Each item in the list is checked against the filter condition. If the condition is true for any item, the playbook proceeds to the next node. (If you do not use the **Split by** field on an output that is a list, then if the condition is true for any item in the list, the entire list moves forward to the next node.)
```
Comment (jpipkin1) [inline docs/cloud-soar/automation.md:285]:
```suggestion
1. (Optional) Use **Split by** to select an output if it is a list (array) and you want to evaluate each item separately. Each item in the list is checked against the filter condition. If the condition is true for an item, the item is passed to the next node. (If you do not use the **Split by** field on an output that is a list, then if the condition is true for any item in the list, the entire list moves forward to the next node.)
```

File: docs/platform-services/automation-service/automation-service-playbooks.md [hunk 4]
```diff
@@ -183,6 +183,7 @@ A filter node filters results from the preceding action based on the condition y
 1. [Add an action node](#add-an-action-node-to-a-playbook). 
 1. Hover your mouse over an action node and click the **+** button. The available nodes are displayed. <br/><img src={useBaseUrl('img/platform-services/automation-service/automations-add-filter-node.png')} alt="Add filter node" style={{border:'1px solid gray'}} width="500"/>
 1. Click **Filter**. The filter node configuration dialog displays. <br/><img src={useBaseUrl('img/platform-services/automation-service/automations-add-filter-node-conditions.png')} alt="Add filter node conditions" style={{border:'1px solid gray'}} width="500"/>
+1. (Optional) Use **Split by** to select an output if it is a list (array) and you want to evaluate each item separately. Each item in the list is checked against the filter condition. If the condition is true for any item, the playbook proceeds to the next node. (If you do not use the **Split by** field on an output that is a list, then if the condition is true for any item in the list, the entire list moves forward to the next node.)
```
Comment (jpipkin1) [inline docs/platform-services/automation-service/automation-service-playbooks.md:186]:
```suggestion
1. (Optional) Use **Split by** to select an output if it is a list (array) and you want to evaluate each item separately. Each item in the list is checked against the filter condition. If the condition is true for an item, the item is passed to the next node.  (If you do not use the **Split by** field on an output that is a list, then if the condition is true for any item in the list, the entire list moves forward to the next node.)
```

**Pull request merged**